src/pages/home/home.ts

```diff
@@ -136,10 +139,51 @@ export class HomePage {
     };
   }
 
+  private async showNewFeatureSlides() {
+    if (this.appProvider.isLockModalOpen) return;
+    const disclaimerAccepted = this.profileProvider.profile.disclaimerAccepted;
+    // console.log(`>>> Disclaimer Accepted ${disclaimerAccepted}`);
+    // const onboardingState = await this.persistenceProvider.getOnboardingFlowFlag();
+    if (!disclaimerAccepted) {
+      // first time using the App -> don't show
+      this.persistenceProvider.setNewFeatureSlidesFlag(
+        this.appProvider.info.version
+      );
+      return;
+    }
+    this.persistenceProvider.getNewFeatureSlidesFlag().then(value => {
+      if (!value || value !== this.appProvider.info.version) {
+        const feature_list = this.newFeatureData.get();
+        if (feature_list && feature_list.features.length > 0) {
+          const modal = this.modalCtrl.create(NewFeaturePage, {
+            featureList: feature_list
+          });
+          modal.present();
+          modal.onDidDismiss(data => {
+            if (data) {
+              if (typeof data === 'boolean' && data === true) {
+                this.persistenceProvider.setNewFeatureSlidesFlag(
+                  this.appProvider.info.version
+                );
+              } else if (typeof data !== 'boolean') {
+                this.events.publish('IncomingDataRedir', data);
+              }
+            }
+          });
+        } else {
+          this.persistenceProvider.setNewFeatureSlidesFlag(
+            this.appProvider.info.version
+          );
+        }
+      }
+    });
+  }
+
   async ionViewWillEnter() {
     const config = this.configProvider.get();
     this.totalBalanceAlternativeIsoCode =
       config.wallet.settings.alternativeIsoCode;
+    await this.showNewFeatureSlides();
```

Comment:
I think it's not necessary to wait for that modal instead to continue executing the next code. I should remove `await` in this case.